tests/btcproxy.py

```diff
@@ -93,6 +93,7 @@ def stop(self):
         self.server.stop()
         self.proxy_thread.join()
         logging.debug("BitcoinRpcProxy shut down after processing {} requests".format(self.request_count))
+        self.rpcport = 0
```

Comment:
What's this change for?

Reply:
This is indeed a leftover from a hack. I'll fix it and rebase.